### ISSUE TITLE
Choose database is empty and can't retrieve KeePassSync options

### ISSUE THREAD

**Issue by papa-smurf** [mitchcapper/KeePassSync#1]
On Keepass 2.56 (Windows, 64-bit) with KeePassSync 8.0:

The `Choose database` window that pops up after entering your S3 details stays empty. It does show a database after I manually upload it to the relevant S3 bucket myself, which is not a part of [the documentation](https://github.com/mitchcapper/KeePassSync/blob/master/AMAZON_S3.md)

After I manually add it I can select it and open it, but I can't sync it and keep getting the following message:

![image](https://github.com/mitchcapper/KeePassSync/assets/4224975/7e0d4c0d-3794-48e2-8bb8-a6683079761e)

It doesn't matter if I select `Overwrite if Newer` or `Synchronize` or any other merge option, the result stays the same and the sync button is also grayed out:

![image](https://github.com/mitchcapper/KeePassSync/assets/4224975/ef20d819-7808-40ce-894c-e7649ebe6e03)

What am I missing?

**Comment by mitchcapper** [comment]
Well the "Open From" provider is meant to allow you to open a database that already exists in the cloud.  Once yo do I think you would have to save it locally but I don't honestly know what KP does with it at that point.  It is a feature I don't think gets a lot of use.

I would try the following for using KeePassSync just to make sure everything works and nothing overrides something important on the first try:

Create a new bucket in your S3 storage provider.
Create a new database in keepass open that database and go to show option in KeePassSync.  Now enter the S3 info for where you want it to sync to (under edit keepass entry under "account information").  Then save the database, you should see at the bottom it say Synchronizing and then Synchronized a few minutes later.

Now every time you save the DB it will auto sync to S3.

Now for an existing database you want to start syncing to S3 you do the same as above but rather than creating a new database you would just open your existing, go to KeePassSync options and again enter the account info.


If you wanted to open a database from S3 that was not already on your PC I think the steps would be:
1) Use the Open in KeePassSync when you dont have a database already open in keepass, you enter your S3 creds and it shows a list of potential ones to open
2) Once it opens you likely need to save it to somewhere locally
3) Go into KeePassSync options (not open again) and re-enter your S3 info here that you entered in #1.   

I guess a better option would be for us to offer to allow you to save the info you entered in step 1 into the database once you opened it so you don't have to re-enter it in S3.   I just don't think it comes up often enough to be worth it for most users.   If you uploaded the database in the first place to S3 using KeePassSync then the database would already have a keepass entry containing the S3 sync data it would use after that initial open.



So in short:
Ignore the "Open" option in KeePassSync, it is there as that is an option KeePass supports for storage plugins and has limited use, but is not the primary use for KeePassSync.
Open an existing database, go to "Show Options" select S3 and click edit on the keepass entry under the account info box.  
Save and it should sync:)
